forward_script/src/QueryBuilder.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+
+namespace Wikimedia\ForwardScript;
+
+class QueryBuilder {
+
+	public function getQuery( $info, $pageName, $id = '', $coordinates = [] ) {
+		$category = preg_replace( '/^(:?Category|Kategorie):/', '', $info->category );
+		$query = array_merge( array_filter( $coordinates ), [ 'categories' => $category ] );
+		if ( $id && empty( $info->duplicate_ids ) && empty( $info->id_not_found ) ) {
```

Comment:
The script either returns `{"id_not_found":True}` or a more complex structure, containing `duplicate_ids` as one of the values. So the condition could be rearranged like this:
```php
if ( $id && !isset( $info->id_not_found ) && !$info->duplicate_ids )
```
Would that be better?